npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -49,16 +45,7 @@ export const CYPRESS_WEBPACK_ENTRYPOINT = path.resolve(__dirname, 'browser.js')
  */
 function modifyWebpackConfigForCypress (webpackConfig: Partial<Configuration>) {
   if (webpackConfig?.plugins) {
-    webpackConfig.plugins = webpackConfig.plugins.filter((plugin) => {
-      if (removeList.includes(plugin.constructor.name)) {
-        /* eslint-disable no-console */
-        console.warn(`[@cypress/webpack-dev-server]: removing ${plugin.constructor.name} from configuration.`)
```

Comment:
We modify the webpack config pretty heavily but don't console log for every tweak we make. Removing this log so as to clean up the terminal output.